### PR TITLE
Markdown Improvement

### PR DESCRIPTION
**Pull request by ckouder**
- add html tag supports #148
- XSS safe #157 
- using `vue-showdown` instead of `vue-simple-markdown`

### CODE REVIEW

File: plugins/markdown.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
-Vue.use(VueSimpleMarkdown)
+const NewVueShowdown = {
+    ...VueShowdown,
+    computed: {
+        ...VueShowdown.computed,
+        outputHtml() {
+            if (this.converter) {
+                this.converter.setFlavor(this.flavor || 'allOn')
+                const html = this.converter.makeHtml(this.inputMarkdown)
+                return sanitizeHtml(html)
```
Comment (carboxylman) [inline plugins/markdown.js:13]: My thought here is that we want to also disallow relative `a` `href` attrs.  Can we use the `allowProtocolRelative: false` attr (https://www.npmjs.com/package/sanitize-html#default-options) to accomplish that?

**Review by carboxylman — CHANGES_REQUESTED** [review]

File: plugins/markdown.js (hunk 1 above)
Comment (ckouder) [reply plugins/markdown.js:13]: Unfortunately no... this filter only filters out links begin with `//`. Here is a reference: https://en.wikipedia.org/wiki/Wikipedia:Protocol-relative_URL#:~:text=about%2Fhistory%2F)-,Definition,t%20support%20HTTPS%20at%20all. We need customized filter to kick out relative hrefs

Comment (ckouder) [reply plugins/markdown.js:13]: Why do we need to disallow it?

Comment (carboxylman) [reply plugins/markdown.js:13]: On further reflection, I guess there isn't a use case for worrying about relative URLs here.

**Review by carboxylman — APPROVED** [review]

**Pull request merged**